Cargo.toml

```diff
@@ -64,6 +65,12 @@ usb-device = "0.2.3"
 usbd-serial = "0.1.0"
 
 [features]
+# Default features
+default = ["rtc"]
+
+# Support for RTC, pulls in rtcc and chrono dependencies
+rtc = ["rtcc"]
```

Comment:
Is the added complexity of having another Cargo feature worth the flexibility it provides? I'm not sure, and personally I would probably default to making the RTC and the `rtcc` dependency non-optional, unless that causes problems in practice.

But I think it's fine, definitely not a blocker.

Reply:
I don't think that's a lot of added complexity, since it's a single `#[cfg(...)]`. On the other hand, it enables people to reduce the number of dependencies they use, in case RTC is not used in a firmware project.

(Regarding binary size it probably doesn't make a relevant difference in case the functionality from RTC isn't actually used, right?)